src/scss/Main.scss

```diff
@@ -15,3 +15,7 @@ p {
   color: black;
   margin: 0.4em auto;
 }
+
+.d-flex {
```

Comment:
we might want to have our own bootstrap.scss file instead and lop classes like these into that file. It should be only used for dev purposes.